ITEM.tsv

```diff
@@ -5828,41 +5828,41 @@ ITEM_20150717_005827	Kingdom Army Guard's Breastplate
+ITEM_20150729_005848	New possibilities have been discovered for weaponcrafting thanks to the emergence of new monsters since the Medis Diena.
+ITEM_20150729_005849	This weapon used to be only obtainable by clerics with permission from the royal household. Such restrictions have become meaningless since the Medis Diena.
 ITEM_20150729_005850	Minotaur Shield
-ITEM_20150729_005851	{memo X}The recipe for this shild has only been discovered recently. It may not be possible to craft it after the Day of Divine Tree.
+ITEM_20150729_005851	{memo X}The recipe for this shild has only been discovered recently. It may not be possible to craft it after the Medis Diena.
 ITEM_20150729_005852	Since the Day of Goddess, many recipes were found through the use of altenative ingredients from monsters.
-ITEM_20150729_005853	Crafters say that it has becom easier to empower a wooden wand's attributes since the Day of Divine Tree.
-ITEM_20150729_005854	Prior to the Day of Divine Tree, there was nobody who whould use this weapon because it used Demonicpowers, However, with the vanishing of the Goddesses, desperate people have begun using it.
+ITEM_20150729_005853	Crafters say that it has becom easier to empower a wooden wand's attributes since the Medis Diena.
+ITEM_20150729_005854	Prior to the Medis Diena, there was nobody who whould use this weapon because it used Demonicpowers, However, with the vanishing of the Goddesses, desperate people have begun using it.
 ITEM_20150729_005855	Rumor has it that supplies of leather increased after Medis Diena.
-ITEM_20150729_005856	One of the benefits of the Day of Divine Tree is that crafters have found a way to reuse monsters's remains for stronger armor.
-ITEM_20150729_005857	Since the Day of Divine Tree, many equipment crafted from new material, like these pants, are replacing traditional ones.
-ITEM_20150729_005858	Standard armor given to the Royal Guard prior to the Day of Divine Tree.
-ITEM_20150729_005859	Since their assumption of office prior to the Day of Divine Tree, the Cryomancer Master has stopped the tradition of awarding this pendant to people. Now this pendant must be either obtained or crafted by users themselves.
-ITEM_20150729_005860	Many new magic items that use the power of monsters have been invented since the Day of Divine Tree.
+ITEM_20150729_005856	One of the benefits of the Medis Diena is that crafters have found a way to reuse monsters's remains for stronger armor.
+ITEM_20150729_005857	Since the Medis Diena, many equipment crafted from new material, like these pants, are replacing traditional ones.
+ITEM_20150729_005858	Standard armor given to the Royal Guard prior to the Medis Diena.
+ITEM_20150729_005859	Since their assumption of office prior to the Medis Diena, the Cryomancer Master has stopped the tradition of awarding this pendant to people. Now this pendant must be either obtained or crafted by users themselves.
```

Comment:
his assignment to office?
It's talking about one Cryomancer Master. Dunno his/her gender